src/ZuluSCSI_cdrom.cpp

```diff
@@ -1345,9 +1339,10 @@ static void doReadSubchannel(bool time, bool subq, uint8_t parameter, uint8_t tr
             *buf++ = (lba >= trackinfo.data_start) ? 1 : 0; // Index number (0 = pregap)
             if (time)
             {
+                *buf++ = 0;
                 LBA2MSF(lba, buf);
                 dbgmsg("------ ABS M ", *(buf+1), " S ", *(buf+2), " F ", *(buf+3));
```

Comment:
I think this debug message and line 1360 below may have off-by-one error now. Wouldn't it be buf+0..buf+2?

Reply:
Ah, definitely. Fixed, thanks for the catch!